kubelet/datadog_checks/kubelet/prometheus.py

```diff
@@ -25,10 +25,6 @@ def __init__(self, check):
 
         self.NAMESPACE = 'kubernetes'
 
-        self.metrics_mapper = {
-            'kubelet_runtime_operations_errors': 'kubelet.runtime.errors',
-        }
-
```

Comment:
This is actually collected from `/metrics` and not `/metrics/cadvisor` so its been moved to the `metrics_mapper` for kubelet metrics.